README.md

```diff
@@ -155,16 +150,126 @@ Wait until a job is complete and then download its output:
 
 ```py
 # Local path to which to download the output
-job_output_path = "/path/to/output.zip"
+output_path = "/path/to/output.zip"
 
 api.wait_until_job_completes(job_id)
-api.download_job_output(job_id, job_output_path)
+api.download_job_output(job_id, output_path)
 ```
 
 Get the status of a job:
 
 ```py
-pprint(api.get_job_status(job_id))
+status = api.get_job_status(job_id)
+```
+
+
+## Application Quickstart
+
+This section provides a brief guide to using the Vision Services API with your
+application admin account.
```

Comment:
This relates to what we briefly touched upon Friday. I think we are being misleading by suggesting this is using the application admin's account. This is really a developer private key (token if you want) only for the application itself. The application admin's account is solely responsible for logging into the admin dashboard on the platform. Two linked but separate concepts.

Reply:
Removed the notion of "app admin".